### PR TITLE
improve README, move bulk of info to main CrateDB docs

### PR DESCRIPTION
**Pull request by nomicode**
see comments and working on docker-library/docs#891

planning doc: https://trello.com/c/BcRUbPFR/25

### CODE REVIEW

File: crate/get-help.md [hunk 1]
```diff
@@ -1 +1,18 @@
-[the public Crate community on Slack](https://crate.io/docs/support/slackin/)
+## Issues
+
+Please report issues via the [GitHub issue tracker](https://github.com/crate/docker-crate/issues).
```
Comment (chaudum) [inline crate/get-help.md]: I would also link the crate/crate repo
Docker related issues should be reported to crate/docker-crate, database issues to crate/crate

**Review by chaudum — CHANGES_REQUESTED** [review]
otherwise ok!

**Review by chaudum — APPROVED** [review]

**Comment by nomicode** [comment]
@chaudum some fix ups per feedback. can you re-ack?

**Review by chaudum — APPROVED** [review]

**Comment by nomicode** [comment]
merged in upstream

**Pull request closed without merging**